packages/headless/src/composables/comboBox/createComboBox.ts

```diff
@@ -1,50 +1,85 @@
-import { computed, ref, type Ref } from "vue";
+import { computed, ref, type MaybeRef, type Ref } from "vue";
 import { createBuilder } from "../../utils/builder";
 import { createId } from "../../utils/id";
+import { createListbox } from "../listbox/createListbox";
+
+export type CreateComboboxOptions<TValue extends string> = {
```

Comment:
Would prefer to use `Pick<CreateListboxOptions>` for the properties like `onActivateFirst` etc. that are passed through to the listbox

Reply:
done